scripts/alert-missing-dumps.sh

```
@@ -1,49 +1,121 @@
+#   - `mc ls` colunas: [data hora UTC] tamanho classe arquivo → sort -k1,2
+#   - comparações de data como string "YYYY-MM-DD HH:MM:SS" (ordem lexicográfica
+#     = ordem cronológica neste formato)
+# ═══════════════════════════════════════════════════════════════════════════
+set -u
 
-set -euo pipefail
+R2_BUCKET="${R2_BUCKET:-promo-brindes-backups}"
+MAX_AGE_HOURS="${MAX_AGE_HOURS:-26}"
+STATE_DIR="${STATE_DIR:-/state}"
+TODAY=$(date -u +%Y-%m-%d)
+export PGPASSWORD="$(cat /run/secrets/supabase_db_password_v1 2>/dev/null || echo x)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)